src/components/App/index.jsx

```diff
@@ -101,8 +102,9 @@ const AppWrapper = () => {
       return false;
     }
     const startTimestamp = config.MAINTENANCE_ALERT_START_TIMESTAMP;
-    if (startTimestamp) {
-      return new Date() > new Date(startTimestamp);
+    const stopTimestamp = config.MAINTENANCE_ALERT_STOP_TIMESTAMP;
+    if (startTimestamp && stopTimestamp) {
```

Comment:
[suggestion] I think this should still support having a `startTimestamp` with no configured `stopTimestamp`, for scenarios where we might not know the expected duration of a maintenance window. For example:

* Both `startTimestamp` and `stopTimestamp` are defined, then the check for whether today's date is between the two start/stop dates make sense.
* Only `startTimestamp` defined, the date check should only check whether today's date is after the start.
* Only `endTimestamp` defined, the date check should only check whether today's date is before the end.